org.lflang/src/org/lflang/federated/launcher/FedCLauncher.java

```diff
@@ -75,28 +75,28 @@ String compileCommandForFederate(FederateInstance federate) {
             errorReporter.reportError("Failed to create file config for federate "+federate.name);
             return "";
         }
-        
+
         String commandToReturn = "";
-        // FIXME: Hack to add platform support only for linux systems. 
+        // FIXME: Hack to add platform support only for linux systems.
```

Comment:
Could you please create an issue for this and reference it in the code?

Reply:
Done: #1418